### PR TITLE
samples: mss: Minor tweaks

### PR DESCRIPTION
**Pull request by glarsennordic**
Tweak PR covering two minor tickets: IRIS-7458 and IRIS-9197

### CODE REVIEW

**Review by jayteemo — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -596,6 +596,9 @@ Cellular samples
     * Handling of L4 disconnect where CoAP connection is paused and socket is kept open, then resumed when L4 reconnects.
     * Checking in CoAP FOTA and shadow polling threads to improve recovery from communications failures.
     * Sysbuild overlays for Wi-Fi and external-flash builds.
+    * CONFIG_TEST_COUNTER_MULTIPLIER to multiply the number of test counter messages sent, for testing purposes.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:599]:
```suggestion
    * The :kconfig:option:`CONFIG_TEST_COUNTER_MULTIPLIER` Kconfig option to multiply the number of test counter messages sent, for testing purposes.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by plskeggs — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (glarsennordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:599]: Ahh, good catch

**Review by peknis — APPROVED** [review]

**Comment by glarsennordic** [comment]
Added DNM: This breaks CoAP due to JWT requiring PicoLib, needs further investigation

**Comment by glarsennordic** [comment]
One of these is blocked on an issue with hostap, so I will split into two PRs.

**Pull request closed without merging**